config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-release-1-5.yaml

```
@@ -153,23 +167,25 @@ presubmits:
         resources:
           requests:
             cpu: 7300m
+            memory: 9Gi
+          limits:
+            cpu: 7300m
+            memory: 9Gi
```

```suggestion
            memory: 32Gi
          limits:
            cpu: 7300m
            memory: 32Gi
```
(e2e test job should use the same resources as the e2e periodics)